docs/sql-ref-datatypes.md

```diff
@@ -314,3 +314,128 @@ SELECT COUNT(*), c2 FROM test GROUP BY c2;
 |        3| Infinity|
 +---------+---------+
 ```
+
+### Type conversion
+
+In general, an expression can contain different data types, type conversion is the transformation of some data types into others in order to solve the expressions. 
```

Comment:
`... can contain different data types, type conversion ...` -> `... can contain different data types and type conversion ...`?

Reply:
`in order to solve the expressions.` -> `in order to resolve type mismatches.`?